cmake/FindUSB1.cmake

```diff
@@ -53,5 +53,4 @@ else (LIBUSB_INCLUDE_DIR AND LIBUSB_LIBRARIES)
   FIND_PACKAGE_HANDLE_STANDARD_ARGS(LIBUSB DEFAULT_MSG LIBUSB_LIBRARIES LIBUSB_INCLUDE_DIR)
 
   MARK_AS_ADVANCED(LIBUSB_INCLUDE_DIR LIBUSB_LIBRARIES)
-
-endif (LIBUSB_INCLUDE_DIR AND LIBUSB_LIBRARIES)
\ No newline at end of file
+ENDIF(LIBUSB_INCLUDE_DIR AND LIBUSB_LIBRARIES)
```

Comment:
Please have consistence upppercase and lowercase. Line 14 and 19 are lowercase, then suddenly the endif() for both become uppercase

Reply:
I will fix it